tools/v8_gypfiles/v8.gyp

```diff
@@ -271,18 +271,6 @@
           # to appear before libv8_snapshot.a so it's listed explicitly.
           'dependencies': ['v8_base', 'v8_init', 'v8_nosnapshot'],
         }],
-        ['v8_use_snapshot==1 and v8_use_external_startup_data==0', {
-          # The dependency on v8_base should come from a transitive
-          # dependency however the Android toolchain requires libv8_base.a
-          # to appear before libv8_snapshot.a so it's listed explicitly.
-          'dependencies': ['v8_base', 'v8_snapshot'],
```

Comment:
Sorry, my mistake, I should've spotted this. I'd rewrite lines 268-273 to this:
```python
# The dependency on v8_base should come from a transitive
# dependency however the Android toolchain requires libv8_base.a
# to appear before libv8_snapshot.a so it's listed explicitly.
['v8_use_snapshot==1', {
  'dependencies': ['v8_base', 'v8_snapshot'],
}, {
  'dependencies': ['v8_base', 'v8_init', 'v8_nosnapshot'],
],
```

Reply:
I'm sorry I rushed through this.

I don't exactly understand now that I've looked more carefully.

It sets `dependencies` to `['v8_base', 'v8_external_snapshot']` no matter what the value of `want_separate_host_toolset` is? Why was that *in* the condition then?

Also, where do `'v8_init', 'v8_nosnapshot'` come from in the snippet you proposed?

Reply:
> It sets dependencies to ['v8_base', 'v8_external_snapshot'] no matter what the value of want_separate_host_toolset is? Why was that in the condition then?

Probably an oversight.

> Also, where do 'v8_init', 'v8_nosnapshot' come from in the snippet you proposed?

That's copied from line 272.

Reply:
> Probably an oversight.

😅

> That's copied from line 272.

Of course! Got it now. Making the change.